### DISCUSSION TITLE
BilateralFilter importing error

### DISCUSSION THREAD

**Discussion by saruarlive** [Project-MONAI/MONAI#1458] [Q&A]
Hi, @Nic-Ma  et al,
How can I use the bilateral filter as it is not imporing? Could you kindly look into that?

![image](https://user-images.githubusercontent.com/24542592/104142757-2fc07b00-53bd-11eb-80f1-e907e86ee2fc.png)

**Comment by Nic-Ma** [comment]
Hi @saruarlive ,

Thanks for your interest here.
`BilateralFilter` was added after MONAI v0.4 release, so you want to use it, please update to use master branch code directly.

Thanks.

**Comment by rijobro** [comment]
@saruarlive check this comment for info on getting master branch: https://github.com/Project-MONAI/MONAI/pull/1409#issuecomment-756742166.

**Comment by saruarlive** [comment]
@Nic-Ma , @rijobro ,  Tried with the master branch.
still encountering a problem with importing CPP files,
![image](https://user-images.githubusercontent.com/24542592/104634406-de580a80-56a0-11eb-9a5a-e3e2448ca38d.png)

**Comment by wyli** [comment]
@saruarlive this an experimental feature currently not enabled by default, please install monai with `BUILD_MONAI=1` flag:  `BUILD_MONAI=1 pip install git+https://github.com/Project-MONAI/MONAI#egg=MONAI` https://docs.monai.io/en/latest/installation.html#option-2-editable-installation

also many thanks for reporting the issues, next time please post it in the discussion section: https://github.com/Project-MONAI/MONAI/discussions as this is a usage question. Posting some logs in plain text is better than the screenshots, as the screenshots are not searchable

**Comment by tvercaut** [comment]
It might be good to also mention the
```
BUILD_MONAI=1 pip install git+https://github.com/Project-MONAI/MONAI#egg=MONAI
```
option in the root README:
https://github.com/Project-MONAI/MONAI#installing-the-master-branch-from-the-source-code-repository

**Comment by rijobro** [comment]
Or alternatively remove all installation instructions from the README, leaving just the link to https://docs.monai.io/en/latest/installation.html? Would save us from keeping the two in sync with each other.
